packages/abis/bin/mergeAbi.ts

```diff
@@ -0,0 +1,78 @@
+import { ethers } from 'ethers'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+import { Fragment, InterfaceAbi } from 'ethers'
+const abisToMerge = ['RelayPool']
+
+const pastFolder = path.join(__dirname, '..', 'versions')
+const srcFolder = path.join(__dirname, '..', 'src', 'abis')
+
+export const mergeAbis = (
+  currentAbi: InterfaceAbi,
+  previousAbi: InterfaceAbi
+): InterfaceAbi => {
+  const { fragments: current } = new ethers.Interface(currentAbi)
+  const { fragments: previous } = new ethers.Interface(previousAbi)
+
+  const removedFragments: number[] = []
```

Comment:
There is no `currentAbi` or `previousAbi`. Just many ABI files that need to be combined **at runtime** in a single one.